designcompose/src/main/java/com/android/designcompose/DesignText.kt

```diff
@@ -244,10 +249,11 @@ internal fun DesignText(
 
     val textBrushAndOpacity =
         style.node_style.text_color.asBrush(document, density.density, variableState)
+
     val textStyle =
         @OptIn(ExperimentalTextApi::class)
         TextStyle(
-            brush = customBrush ?: textBrushAndOpacity?.first,
+            brush = customBrush ?: textBrushAndOpacity?.first ?: SolidColor(Color.Transparent),
```

Comment:
What does this change solve?

Reply:
When the brush is null, it also draws the text black. So I have to set it to transparent color to draw nothing visually.